README.md

```diff
@@ -90,5 +90,5 @@ For __v1__ I rewrote and redesigned the core library essentially from scratch, a
 * Grailbio [go-dicom](https://github.com/grailbio/go-dicom) -- commits from their fork were applied to ours
```

Comment:
I see the file's permissions have changed from 755 to 644. This is reasonable, but is this intentional?

Reply:
nvm, I see your commit. Anyway, no big deal, this is fine 👍🏽